bin/shorebird.bat

```diff
@@ -18,4 +18,4 @@ SET PowerShellScriptPath=%CurrentDirectory%shorebird.ps1
 
 REM Pass up to four arguments to the PowerShell script
 REM This number is arbitrary and can be increased if needed
-& %powershell_executable% -NoProfile -ExecutionPolicy Bypass -Command "& '%PowerShellScriptPath%' %1% %2% %3% %4%";
+%powershell_executable% -NoProfile -ExecutionPolicy Bypass -Command "& '%PowerShellScriptPath%' %1 %2 %3 %4";
```

Comment:
Don't we currently have more than 4 arguments sometimes?  Is there a way to increase this at least a few?  Or detect someone tried to pass more than 4 and error?

Reply:
Bumped it up to nine